packages/create-react-native-library/src/index.ts

```diff
@@ -8,8 +8,10 @@ import spawn from 'cross-spawn';
 import validateNpmPackage from 'validate-npm-package-name';
 import githubUsername from 'github-username';
 import prompts, { PromptObject } from './utils/prompts';
+import { generateRNApp } from './utils/exampleAppGenerators';
 
 const FALLBACK_BOB_VERSION = '0.18.3';
+const REACT_NATIVE_VERSION = '0.70.0';
```

Comment:
OK, so we won't be specifying any versions to `react-native init`

Reply:
I think it'd also be good to take a `--react-native-version` parameter to force a specific version if the user needs (maybe helpful to test RC versions and in case something broke with latest version)